sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3352,7 +3352,23 @@ object GpuOverrides extends Logging {
             TypeSig.DATE)),
         Some(RepeatingParamCheck("step", TypeSig.integral, TypeSig.integral + TypeSig.CALENDAR))),
       (a, conf, p, r) => new GpuSequenceMeta(a, conf, p, r)
-    )
+    ),
+    expr[BitLength](
+      "The bit length of string data",
+      ExprChecks.unaryProject(
+        TypeSig.INT, TypeSig.INT,
+        TypeSig.STRING, TypeSig.STRING + TypeSig.BINARY),
+      (a, conf, p, r) => new UnaryExprMeta[BitLength](a, conf, p, r) {
+        override def convertToGpu(child: Expression): GpuExpression = GpuBitLength(child)
+      }),
+    expr[OctetLength](
+      "The bit length of string data",
```

Comment:
Sorry one more thing. This is the **byte** length not the **bit** length.

Reply:
Done.